cypress/e2e/tests/pages/explorer/dashboard/cluster-dashboard.spec.ts

```diff
@@ -8,6 +8,7 @@ import { WorkloadsDeploymentsListPagePo } from '@/cypress/e2e/po/pages/explorer/
 import { NodesPagePo } from '@/cypress/e2e/po/pages/explorer/nodes.po';
 import { EventsPagePo } from '@/cypress/e2e/po/pages/explorer/events.po';
 import * as path from 'path';
+import { eventsNoDataset } from '@/cypress/e2e/blueprints/explorer/cluster/events';
 import { generateEventsDataLarge, generateEventsDataSmall } from '@/cypress/e2e/blueprints/explorer/cluster/events';
```

Comment:
this can be 
> import { generateEventsDataLarge, generateEventsDataSmall, eventsNoDataset } from '@/cypress/e2e/blueprints/explorer/cluster/events';

Reply:
Good catch I just went with the auto generated merge.